docs/documentation.md

```diff
@@ -17,6 +17,124 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
+# RabbitMQ Documentation
+
+Welcome to RabbitMQ documentation. Use the drop-down menu on the top right of any page to select the documentation version for the RabbitMQ release that you are using. We update each version of documentation with the latest minor and patch information for that release.
+
+For previous 3.12.x documentation, select the **3.12** option from the drop-down menu. 3.12.x documentation is hosted on our previous RabbitMQ site. This documentation is not versioned. 
+
+Use the navigation on the left to browse through documentation for your release of RabbitMQ.
+
+## User: Developer
+
+**Important:** The following table is a generic list of tasks a developer would follow to use RabbitMQ in an application. It is important to note that the direction a developer can take is very dependent on how they want use RabbitMQ in an application. For example, the way to open a connection differs a lot depending on the messaging protocol a developers uses. This is just one factor, there are many more. 
+
+<table id="developer">
+ <tr>
+    <th class="centre">Task</th>
+    <th class="centre">Documentation</th>
+  </tr>
+ <tr>
+    <td class="centre">Start by going through the tutorials</td>
+    <td class="centre"><a href="https://www.rabbitmq.com/tutorials">Getting Started</a></td>
+  </tr>
+  <tr>
+    <td class="centre">Choose a messaging protocol</td>
+    <td class="centre"><a href="https://www.rabbitmq.com/protocols.html">What Protocols does RabbitMQ support</a></td>
+  </tr>
+  <tr>
+    <td class="centre">Open a Connection</td>
+    <td class="centre"><a href="https://www.rabbitmq.com/connections.html">Connections</a></td>
+  </tr>
+   <tr>
+   <td class="centre">Publish and Consume Messages </td>
+    <td class="centre">
+    <ul>
+      <li><a href="./publishers.md">Publishing Messages</a></li>
+      <li><a href="./publishers.md">Consuming Message</a></li>
+      <li><a href="./confirms.md">Consumer Acknowledgements and Publisher Confirms</a></li>
+    </ul>
+   </td>
```

Comment:
After looking at this again, the list is very close to what is tought by the tutorials (even though they are AMQP 0-9-1 centric).

Thus I'm wondering if the first step should point to tutorials, then move to intermediate/advanced topics directly.